src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private int radius;
+
+    public Circle(String color, int radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, color: " + getColor() + ", area: "
+                + getArea() + " sq. units, radius: " + radius + " units");
```

Comment:
Use the `name()` method instead of `toString()` when you want to get the String representation of enum constants. If you have an enum for color, make sure to use `color.name()` instead of `getColor()` if `getColor()` is overridden to return something other than the enum constant's name.